pkgs/tools/misc/papis/default.nix

```diff
@@ -1,39 +1,43 @@
 { buildPythonApplication, lib, fetchFromGitHub
 , argcomplete, arxiv2bib, beautifulsoup4, bibtexparser
-, configparser, habanero, papis-python-rofi, pylibgen
-, prompt_toolkit, pyparser, python_magic, pyyaml
-, requests, unidecode, urwid, vobject, tkinter
+, configparser, dmenu-python, habanero, papis-python-rofi
+, pylibgen, prompt_toolkit, pyparser, pytest, python_magic
+, pyyaml, requests, unidecode, urwid, vobject, tkinter
 , vim
 }:
 
 buildPythonApplication rec {
   pname = "papis";
-  version = "0.5.2";
+  version = "0.5.3";
 
   # Missing tests on Pypi
   src = fetchFromGitHub {
     owner = "alejandrogallo";
```

Comment:
Github redirects from `https://github.com/alejandrogallo/papis` to `https://github.com/papis/papis` so this should be updated.